examples/lb3-application/lb3app/common/models/coffee-shop.js

```diff
@@ -7,6 +7,7 @@
 
 const debug = require('debug')('loopback:example:lb3application');
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 module.exports = function (CoffeeShop) {
```

Comment:
IMO, our naming-convention rules should allow anonymous (unnamed) functions. Maybe this is a bug in naming-convention rule? I am fine investigate this out of scope of this pull request.

Reply:
Strangely enough, I am not able to reproduce this problem in a sandbox project, so that I could report an issue to typescript-eslint project. Something weird is going on with our eslint setup.

Reply:
You probably misread the violation. The complaint is against the usage of `CoffeeShop` as the parameter name, which is not camelCase.